src/hooks/useCarbonInit.ts

```diff
@@ -130,6 +130,7 @@ export const useCarbonInit = () => {
       if (isBlocked && !lsService.getItem('hasSeenRestrictedCountryModal')) {
         openModal('restrictedCountry', undefined);
       }
+      lsService.migrateItems();
```

Comment:
migration should happen prior to any local storage access.

Reply:
Moved to the top of the init function